frontend/packages/ceph-storage-plugin/integration-tests/tests/1-install/installFlow.scenario.ts

```diff
@@ -134,6 +145,148 @@ if (TEST_PLATFORM === Platform.OCP || (TEST_PLATFORM === Platform.OCS && MODE ==
   });
 }
 
+if (
+  TEST_PLATFORM === Platform.OCP ||
+  (TEST_PLATFORM === Platform.OCS && MODE === Mode.ATTACHED_DEVICES)
+) {
+  describe('Test creation of Attached Storage Cluster', () => {
+    it('Info Alert should be shown as LSO will not be installed', async () => {
+      await Installer.createAttachedStorageCluster();
+      const msg = await currentSelectors.LSOAlert.getText();
+      expect(msg.includes(LSO_INFO_MSG)).toBe(true);
+    });
+
+    it(
+      'tests subscription flow for LSO Operator',
+      async () => {
+        await Installer.subscribeToLSOOperator();
+      },
+      3 * MINUTE,
+    );
+
+    it('Create Storage Class Wizard should be present once LSO is installed and no storage class is present', async () => {
+      await Installer.goToInstalledOperators();
+      await Installer.createAttachedStorageCluster();
+      expect(currentSelectors.LSOWizard.isPresent()).toBeTruthy();
+    });
+
+    it('Create Storage Cluster View should be present when LSO is installed and storage class is present', async () => {
+      execSync(`echo '${JSON.stringify(storageClass)}' | oc apply -f -`);
+      // need to wait for some time in order to reflect the storage class
+      await Installer.goToInstalledOperators();
+      await Installer.createAttachedStorageCluster();
+      // wizrad should not be present
+      expect(currentSelectors.LSOWizard.isPresent()).toBeFalsy();
+
+      // installation page should not be present
+      const msg = await currentSelectors.LSOAlert.getText();
+      expect(msg.includes(LSO_INFO_MSG)).toBe(false);
+      execSync(`echo '${JSON.stringify(storageClass)}' | oc delete -f -`);
+    });
+
+    it('Should show error message on Create Storage Cluster View, if storage class does not contain minimum 3 nodes ', async () => {
+      execSync(`echo '${JSON.stringify(storageClass)}' | oc apply -f -`);
+      // need to wait for some time in order to reflect the storage class
```

Comment:
Where is this waiting happening? If it's happening inside `goToInstalledOperators` then remove this comment.

Reply:
Should be removed.